src/flows/pipes/Notification/endpoints/Mailgun/index.ts

```diff
@@ -33,37 +35,56 @@ export default register => {
 	|> monitor["notify"](
     data: notification,
     endpoint: ((r) => {
-        apiKey = secrets.get(key: "${data.apiKey}")
-        http.post(
-            url: "https://api.mailgun.net/v3/${data.domain}/messages",
-            headers: {
-              "Content-type": "application/json",
-              "Authorization": "Basic api:\${apiKey}"
-            },
-            data: bytes(v: "{
-              \\"from\\": \\"Username mailgun@${data.domain}\\",
-              \\"to\\": \\"${data.email}\\",
-              \\"subject\\": \\"InfluxDB Alert\\",
-              \\"text\\": \\"\${ r._message }\\"
-            }"))
-    })
-  )`,
-    generateTestQuery: data => `
-    apiKey = secrets.get(key: "{data.apiKey}")
-    http.post(
-        url: "https://api.mailgun.net/v3/${data.domain}/messages",
+      apiKey = secrets.get(key: "${data.apiKey}")
+      auth = http.basicAuth(u: "api", p: "\${apiKey}")
+      url = "https://api.mailgun.net/v3/${data.domain}/messages"
+      data = strings.joinStr(arr: [
+          "from=Username mailgun@${data.domain}",
```

Comment:
Should the `Username` or `Username mailgun` be a variable instead of hard coded string? Looks like anything after `from=` should be an email address

Reply:
this is a valid point. I will make a change. Infact, I think we need to change the name of sender from `Username` to `InfluxDB Mailgun`..

@drdelambre Thoughts?

Reply:
Nice! This makes more sense

Reply:
@subirjolly i have no idea how mailgun works. pop up the mailgun docs and see if the username is configurable

Reply:
updated to just be email